domain/modeldefaults/state/state.go

```diff
@@ -25,10 +25,15 @@ type State struct {
 }
 
 // ConfigDefaults returns the default configuration values set in Juju.
-func (s *State) ConfigDefaults(_ context.Context) map[string]any {
+func ConfigDefaults(_ context.Context) map[string]any {
 	return config.ConfigDefaults()
 }
 
+// ConfigDefaults returns the default configuration values set in Juju.
+func (s *State) ConfigDefaults(ctx context.Context) map[string]any {
+	return ConfigDefaults(ctx)
+}
+
```

Comment:
I'm not sure why either of these methods are necessary, why don't we just inline them?

Reply:
Eventually we will. As we try and move away from the business logic being in `environ/config` we can move the defaults out of that location and back into this package. For the moment however we still have a few call sites for NewConfig with defaults that need to be stomped out before we can. There is JUJU-6138 to do this work at the end.